database/migrations/2022_04_09_120404_create_wallets_table.php

```diff
@@ -15,7 +15,7 @@ public function up(): void
             $table->string('external_id')->unique()->nullable();
             $table->string('verification_id')->unique()->nullable();
             $table->string('public_key')->unique()->nullable();
-            $table->boolean('managed')->default(false)->index();
+            $table->boolean('managed')->default(false);
```

Comment:
I think you'll need to update the schema dump `database/schema/mysql-schema.sql` to avoid any weird errors. 

To update the schema dump you can do these steps
1. Run `php vendor/bin/testbench migrate:fresh` to make sure that db schema is updated.
2. Then run `php vendor/bin/testbench schema:dump`
3. Copy the schema dump from vendor directory `vendor/orchestra/testbench-core/laravel/database/schema/mysql-schema.sql` to `database/schema/mysql-schema.sql`